R/PackageFunctionReporter.R

```diff
@@ -56,48 +56,26 @@ FunctionReporter <- R6::R6Class(
     inherit = AbstractGraphReporter,
 
     public = list(
-        get_summary_view = function(){
-
-            # Calculate network measures if not already done
-            # since we want the node measures in summary
-            invisible(self$network_measures)
-
-            # Create DT for display
-            tableObj <- DT::datatable(
-                data = self$nodes
-                , rownames = FALSE
-                , options = list(
-                    searching = FALSE
-                    , pageLength = 50
-                    , lengthChange = FALSE
-                )
-            )
-            # Round the double columns to three digits for formatting reasons
-            numCols <- names(which(unlist(lapply(tableObj$x$data, is.double))))
-            tableObj <- DT::formatRound(columns = numCols, table = tableObj
-                                        , digits=3)
-            return(tableObj)
-        }
-    ),
 
-    active = list(
-        edges = function(){
-            if (is.null(private$cache$edges)){
-                log_info("Calling extract_network() to extract nodes and edges...")
-                private$extract_network()
-            }
-            return(private$cache$edges)
-        },
-        nodes = function(){
-            if (is.null(private$cache$nodes)){
-                log_info("Calling extract_network() to extract nodes and edges...")
-                private$extract_network()
+        calculate_default_measures = function() {
+            # Calculate test coverage if pkg_path is set and source code available
+            if (!is.null(private$pkg_path)){
+                private$calculate_test_coverage()
             }
-            return(private$cache$nodes)
-        },
+
+            super$calculate_default_measures()
+
+            return(invisible(self))
+        }
+
+    ) # / public
+
+    , active = list(
+
         report_markdown_path = function(){
             system.file(file.path("package_report", "package_function_reporter.Rmd"), package = "pkgnet")
         },
+
```

Comment:
That's because they're not supported by Function Reporter. In any case, I'm going to move those bindings into private methods. I originally did it that way because I wanted to take advantage of active bindings' clean interface. But honestly I don't think users would or should need to use these methods, and they make the interface more complicated.

Reply:
I opened #197 to call out that Function Reporter doesn't support this yet.